### DISCUSSION TITLE
Command returns error straight away without information

### DISCUSSION THREAD

**Discussion by ghost** [community/community#51023] [Actions]
### Select Topic Area

Bug

### Body

My workflow at https://github.com/freent-project/license-translations is used to generate a README for listing translations. However, it won't generate, but rather give an error without a job summary or logs. I also tried restarting the workflow with debugging but nothing still appears.

Here's the code: https://github.com/freent-project/license-translations/blob/a6ebdffef9f2a7d2803cbf92ee2e257d9f0adf7f/.github/workflows/readme-gen.yml

**Comment by airtower-luna** [comment]
Without seeing the output I can't say *why* it's failing either, but I assume the [rather convoluted redirection](https://github.com/freent-project/license-translations/blob/a6ebdffef9f2a7d2803cbf92ee2e257d9f0adf7f/.github/workflows/readme-gen.yml#L13) is eating your log output. I recommend you remove it so the log output goes right to the job log:

```yaml
      - name: Run README generator
        run: python readme_generate.py
```

That way you should be at least able to see what's going wrong.
